zstacklib/zstacklib/utils/lvm.py

```
@@ -2531,6 +2531,18 @@ def disable_multipath():
         raise RetryException("multipath is still running")
 
 
+@bash.in_bash
+def rename_vg(old_vgUuid, new_vgUuid):
+    @linux.retry(times=3, sleep_time=random.uniform(0.1, 1))
```

Ruff (0.14.14)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)